src/Uno.UI/UI/Xaml/Maps/MapControl.cs

```diff
@@ -0,0 +1,727 @@
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"TiltInteractionMode", typeof(MapInteractionMode),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(MapInteractionMode)));
+		public static DependencyProperty TransitFeaturesVisibleProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"TransitFeaturesVisible", typeof(bool),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(bool)));
+		public static DependencyProperty ZoomInteractionModeProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"ZoomInteractionMode", typeof(MapInteractionMode),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(MapInteractionMode)));
+		public static DependencyProperty IsStreetsideSupportedProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"IsStreetsideSupported", typeof(bool),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(bool)));
+		public static DependencyProperty SceneProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"Scene", typeof(MapScene),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(MapScene)));
+		public static DependencyProperty BusinessLandmarksEnabledProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"BusinessLandmarksEnabled", typeof(bool),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(bool)));
+		public static DependencyProperty TransitFeaturesEnabledProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"TransitFeaturesEnabled", typeof(bool),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(bool)));
+		public static DependencyProperty MapProjectionProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"MapProjection", typeof(MapProjection),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(MapProjection)));
+		public static DependencyProperty StyleSheetProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"StyleSheet", typeof(MapStyleSheet),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(MapStyleSheet)));
+		public static DependencyProperty ViewPaddingProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"ViewPadding", typeof(Thickness),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(Thickness)));
+		public static DependencyProperty LayersProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"Layers", typeof(IList<MapLayer>),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(IList<MapLayer>)));
+		public static DependencyProperty RegionProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"Region", typeof(string),
+			typeof(MapControl),
+			new FrameworkPropertyMetadata(default(string)));
+		public IReadOnlyList<MapElement> FindMapElementsAtOffset(global::Windows.Foundation.Point offset)
```

Comment:
Everything that's not implemented should be [NotImplemented]

Reply:
It can't that's thing for this control. The fact that it's implemented or not comes from the `Presenter` part of it.

Reply:
Not sure I follow... I don't see any of those members delegating implementation to the presenter.

Reply:
They will, ultimately, but we don't know which will be in the loaded presenter. At this time, we're probably going to have to manage that at runtime.